services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/taydennys/YhteystietoEntity.kt

```diff
@@ -0,0 +1,41 @@
+package fi.hel.haitaton.hanke.taydennys
```

Comment:
Why put these interfaces in `taydennys` package? Maybe we could have `taydennys` under `hakemus` package and these common interfaces in `hakemus` package?

Reply:
I don't really want to create nested packages like that. We'll end up having half of the codebase inside the `hakemus` package. Like `paatos`, `taydennys`, `muutosilmoitus`, `valmistumisilmoitus`, `allu` and `application.attachment` could all be moved / created under `hakemus`.

I combined the interfaces into a single file and moved it under `hakemus`.